legacy/library/HelloSignLegacy/UnclaimedDraft.php

```diff
@@ -212,6 +212,7 @@ public function toParams()
 
         if (!$this->getClientId()) {
             $except[] = 'client_id';
+            $except[] = 'show_preview';
```

Comment:
here we remove `show_preview` since it is not supported in `unclaimed_draft/create` endpoint.

followed the logic from `Client.php`
```
        // choose url
        $url = $draft->getClientId()
            ? static::UNCLAIMED_DRAFT_CREATE_EMBEDDED_PATH
            : static::UNCLAIMED_DRAFT_CREATE_PATH;
```